src/acquisition/covid_hosp/common/utils.py

```diff
@@ -113,13 +126,20 @@ def merge_by_key_cols(dfs, key_cols):
         Data frames to merge, ordered from earliest to latest.
       key_cols: list(str)
         Columns to use as the index.
+      logger structlog.Logger [optional; default False]
+        Logger to receive messages
 
     Returns a single data frame containing the most recent data for each state+date.
     """
 
     dfs = [df.set_index(key_cols) for df in dfs
            if not all(k in df.index.names for k in key_cols)]
     result = dfs[0]
+    if logger and len(dfs) > 7:
```

Comment:
Yes; [the state-daily source updates every 1-6 days on an irregular schedule](https://cmu-delphi.github.io/delphi-epidata/api/covid_hosp.html#metadata), meaning we expect between 1 and 6 files in a block update; bumped up to 7 because if they did a weekly batch update every once in a while that wouldn't be unreasonable.

Also consistent with the comment a few lines down.